env.md

```diff
@@ -11,6 +11,16 @@ permalink: /env/
 
 If you are using Babel version 7 you will need to run `npm install @babel/preset-env` and have `"presets": ["@babel/preset-env"]` in your configuration.
 
+## Before you do anything
+
+You might end up on this page because you saw a message in the terminal like this:
+
+>🙌  Thanks for using Babel: we recommend using babel-preset-env now: please read babeljs.io/env to update!
+
+Before you proceed further, ask yourself: are you *using* Babel? Check your `package.json` and look for `babel-preset-es2015` or a similar preset there. If you see a preset like this in your `package.json`, read on!
+
+If you don't use Babel or don't use deprecated yearly presets, you probably saw this message because *another package* you depend on uses them. **In that case there's nothing *you* need to do**. Nevertheless, it might be a good idea to find out which package uses the deprecated presets, and help them migrate by sending a pull request. You can find this out by running `npm ls babel-preset-es2015` which will show the dependency tree. 
```

Comment:
I'm not an english expert but I would replace:

> or don't use deprecated yearly presets

or don't use a deprecated yearly preset

Reply:
> find out which package uses the deprecated presets

find out which package uses a deprecated preset

Reply:
Send a follow up please if you don't mind :-) Thanks!

Reply:
Do this only happen with npm, yarn doesn't show the warning? In that case a note about using `yarn why babel-preset-es2015` would be helpful